### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by marraknas**

### CODE REVIEW

**Review by PeterHW963 — COMMENTED** [review]
LGTM!

**Review by ulricolo7 — COMMENTED** [review]
LGTM?

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -445,38 +445,26 @@ testers are expected to do more *exploratory* testing.
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   2. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
```
Comment (XinghuaJulia) [inline docs/DeveloperGuide.md:448]:
```suggestion
   2. Double-click the jar file Expected: Shows the GUI with a set of sample Udders. The window size may not be optimum.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -445,38 +445,26 @@ testers are expected to do more *exploratory* testing.
...
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
-
 ### Deleting a person
```
Comment (XinghuaJulia) [inline docs/DeveloperGuide.md:457]:
```suggestion
### Deleting an Udder
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -445,38 +445,26 @@ testers are expected to do more *exploratory* testing.
...
 
 1. Deleting a person while all persons are being shown
```
Comment (XinghuaJulia) [inline docs/DeveloperGuide.md:459]:
```suggestion
1. Deleting an Udder while all Udders are being shown
```

**Review by XinghuaJulia — COMMENTED** [review]
LGTM! Minor nits on how Person and Contact should be Udders, but otherwise, good job :DDDDDDDD

**Pull request closed without merging**